compiler/rustc_borrowck/src/diagnostics/conflict_errors.rs

```diff
@@ -1306,14 +1307,33 @@ impl<'tcx> MirBorrowckCtxt<'_, '_, '_, 'tcx> {
                 return false;
             }
             inner_expr = inner;
+            let typeck_result = self.infcx.tcx.typeck(self.mir_def_id());
```

Comment:
nit: move that query out of the loop

Reply:
OK, thank your for your swift review. :)